### PR TITLE
chore(store): bump confd to v0.8

### PR DESCRIPTION
**Pull request by bacongobbler**

### CODE REVIEW

File: store/base/templates/ceph.conf [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-osd pool default size = {{ .deis_store_size }}
-osd pool default min_size = {{ .deis_store_minSize }}
-osd pool default pg_num = {{ .deis_store_pgNum }}
-osd pool default pgp_num = {{ .deis_store_pgNum }}
-osd recovery delay start = {{ .deis_store_delayStart }}
+osd pool default size = {{ getv "deis/store/size" }}
+osd pool default min_size = {{ getv "deis/store/minSize" }}
+osd pool default pg_num = {{ getv "deis/store/pgNum" }}
+osd pool default pgp_num = {{ getv "deis/store/pgNum" }}
+osd recovery delay start = {{ getv "deis/store/delayStart" }}
```
Comment (aledbf) [inline store/base/templates/ceph.conf]: missing / from 9 to 13

Comment (bacongobbler) [inline store/base/templates/ceph.conf]: good catch, thanks

**Comment by mboersma** [comment]
Code LGTM.

**Comment by aledbf** [comment]
Code LGTM.

**Pull request merged**